src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -28,6 +28,21 @@
             border: 0;
             padding: 0 0 8px 0;
         }
+
+        #collector-content .authenticators .badge {
+            color: var(--white);
+            display: inline-block;
+            text-align: center;
+            min-width: 60px;
+        }
+
+        #collector-content .authenticators .badge.badge-resolved {
+            background-color: var(--green-500);
+        }
+
+        #collector-content .authenticators .badge.badge-not_resolved {
+            background-color: var(--red-500);
```

Comment:
I like the badge, but red seems "errored", what about using a yellow/orange color instead ?

Reply:
Sure! Screenshots are updated too.